mk/esp32-c3.mk

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: Apache-2.0
+CROSS_PREFIX := riscv32-esp-elf-
+include mk/gcc-config.mk
+
+IDF_PATH ?=
+ESP_IDF_LIB ?=
+
+LDSCRIPTS = \
+	hal/esp32-c3.ld \
+	$(IDF_PATH)/components/esp_rom/esp32c3/ld/esp32c3.rom.api.ld \
+	$(IDF_PATH)/components/esp_rom/esp32c3/ld/esp32c3.rom.ld \
+	$(IDF_PATH)/components/esp_rom/esp32c3/ld/esp32c3.rom.newlib.ld \
+	$(IDF_PATH)/components/soc/esp32c3/ld/esp32c3.peripherals.ld
+
+ifeq ($(RNG),NISTKAT)
+	LIBHAL_SRC = \
+		test/common/nistkatrng.c \
+		test/common/aes.c
+	CPPFLAGS += -Itest/common
+else
+	RNG := NOTRAND
+	LIBHAL_SRC = hal/notrandombytes.c
```

Comment:
Is there and hardware random number generator on this board that we may want to use use?

Reply:
I think so, but will need to look into this later